src/convertToObject.js

```diff
@@ -6,7 +6,15 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString.split(';').reduce((acc, item) => {
+    const [key, value] = item.split(':').map((str) => str.trim());
```

Comment:
The use of `map` here is correct for trimming the strings. Ensure that the variable names used are descriptive and not generic like `arr` or `obj`, as per checklist item 2.